shelley/chain-and-ledger/shelley-spec-ledger-test/bench/Shelley/Spec/Ledger/Bench/Rewards.hs

```diff
@@ -63,7 +64,7 @@ import Cardano.Ledger.Era (Era(Crypto))
 -- | Generate a chain state at a given epoch. Since we are only concerned about
 -- rewards, this will populate the chain with empty blocks (only issued by the
 -- original genesis delegates).
-genChainInEpoch :: EpochNo -> Gen (ChainState B)
+genChainInEpoch :: Arbitrary (Core.Value B) => EpochNo -> Gen (ChainState B)
```

Comment:
Given that `B` is `ShelleyEra`, its value will be `Coin`. So we should be able to use the normal `Coin` generator, surely?